tasks/install.yml

```diff
@@ -103,7 +103,7 @@
   become: true
   become_user: "{{ superset_user }}"
   pip:
-    name: superset
+    name: apache-superset
```

Comment:
@nateMJ could we convert this into a variable with the default value being "apache-superset". I think we might also want to update the default superset version to `0.34.0`.